### DISCUSSION TITLE
interrupt vector AC1_AC on ATtiny1616

### DISCUSSION THREAD

**Discussion by wolfgang6444** [SpenceKonde/megaTinyCore#1183] [Q&A]
What is the symbolic name for AC1_AC i.e. (interrupt_vector Nr. 18) on an ATtiny1616.
In other words what is better than:
ISR(_VECTOR(18)) {
...
}

**Comment by hmeijdam** [comment]
You will find the definition in the [header files](https://github.com/SpenceKonde/megaTinyCore/tree/master/megaavr/extras/ioheaders). in your case the iotn1616.h  file


/* ========== Interrupt Vector Definitions ========== */
/* Vector 0 is the reset vector */

/* CRCSCAN interrupt vectors */
#define CRCSCAN_NMI_vect_num  1
#define CRCSCAN_NMI_vect      _VECTOR(1)  /*  */

/* BOD interrupt vectors */
#define BOD_VLM_vect_num  2
#define BOD_VLM_vect      _VECTOR(2)  /*  */

/* PORTA interrupt vectors */
#define PORTA_PORT_vect_num  3
#define PORTA_PORT_vect      _VECTOR(3)  /*  */

/* PORTB interrupt vectors */
#define PORTB_PORT_vect_num  4
#define PORTB_PORT_vect      _VECTOR(4)  /*  */

/* PORTC interrupt vectors */
#define PORTC_PORT_vect_num  5
#define PORTC_PORT_vect      _VECTOR(5)  /*  */

/* RTC interrupt vectors */
#define RTC_CNT_vect_num  6
#define RTC_CNT_vect      _VECTOR(6)  /*  */
#define RTC_PIT_vect_num  7
#define RTC_PIT_vect      _VECTOR(7)  /*  */

/* TCA0 interrupt vectors */
#define TCA0_LUNF_vect_num  8
#define TCA0_LUNF_vect      _VECTOR(8)  /*  */
#define TCA0_OVF_vect_num  8
#define TCA0_OVF_vect      _VECTOR(8)  /*  */
#define TCA0_HUNF_vect_num  9
#define TCA0_HUNF_vect      _VECTOR(9)  /*  */
#define TCA0_CMP0_vect_num  10
#define TCA0_CMP0_vect      _VECTOR(10)  /*  */
#define TCA0_LCMP0_vect_num  10
#define TCA0_LCMP0_vect      _VECTOR(10)  /*  */
#define TCA0_CMP1_vect_num  11
#define TCA0_CMP1_vect      _VECTOR(11)  /*  */
#define TCA0_LCMP1_vect_num  11
#define TCA0_LCMP1_vect      _VECTOR(11)  /*  */
#define TCA0_CMP2_vect_num  12
#define TCA0_CMP2_vect      _VECTOR(12)  /*  */
#define TCA0_LCMP2_vect_num  12
#define TCA0_LCMP2_vect      _VECTOR(12)  /*  */

/* TCB0 interrupt vectors */
#define TCB0_INT_vect_num  13
#define TCB0_INT_vect      _VECTOR(13)  /*  */

/* TCB1 interrupt vectors */
#define TCB1_INT_vect_num  14
#define TCB1_INT_vect      _VECTOR(14)  /*  */

/* TCD0 interrupt vectors */
#define TCD0_OVF_vect_num  15
#define TCD0_OVF_vect      _VECTOR(15)  /*  */
#define TCD0_TRIG_vect_num  16
#define TCD0_TRIG_vect      _VECTOR(16)  /*  */

/* AC0 interrupt vectors */
#define AC0_AC_vect_num  17
#define AC0_AC_vect      _VECTOR(17)  /*  */

/* AC1 interrupt vectors */
#define AC1_AC_vect_num  18
#define AC1_AC_vect      _VECTOR(18)  /*  */

/* AC2 interrupt vectors */
#define AC2_AC_vect_num  19
#define AC2_AC_vect      _VECTOR(19)  /*  */

/* ADC0 interrupt vectors */
#define ADC0_RESRDY_vect_num  20
#define ADC0_RESRDY_vect      _VECTOR(20)  /*  */
#define ADC0_WCOMP_vect_num  21
#define ADC0_WCOMP_vect      _VECTOR(21)  /*  */

/* ADC1 interrupt vectors */
#define ADC1_RESRDY_vect_num  22
#define ADC1_RESRDY_vect      _VECTOR(22)  /*  */
#define ADC1_WCOMP_vect_num  23
#define ADC1_WCOMP_vect      _VECTOR(23)  /*  */

/* TWI0 interrupt vectors */
#define TWI0_TWIS_vect_num  24
#define TWI0_TWIS_vect      _VECTOR(24)  /*  */
#define TWI0_TWIM_vect_num  25
#define TWI0_TWIM_vect      _VECTOR(25)  /*  */

/* SPI0 interrupt vectors */
#define SPI0_INT_vect_num  26
#define SPI0_INT_vect      _VECTOR(26)  /*  */

/* USART0 interrupt vectors */
#define USART0_RXC_vect_num  27
#define USART0_RXC_vect      _VECTOR(27)  /*  */
#define USART0_DRE_vect_num  28
#define USART0_DRE_vect      _VECTOR(28)  /*  */
#define USART0_TXC_vect_num  29
#define USART0_TXC_vect      _VECTOR(29)  /*  */

/* NVMCTRL interrupt vectors */
#define NVMCTRL_EE_vect_num  30
#define NVMCTRL_EE_vect      _VECTOR(30)  /*  */

#define _VECTOR_SIZE 4 /* Size of individual vector. */
#define _VECTORS_SIZE (31 * _VECTOR_SIZE)
